service.go

```diff
@@ -21,6 +22,7 @@ type orgsService interface {
 	getOrgs() ([]orgLink, bool)
 	getOrgByUUID(uuid string) (org, bool, error)
 	isInitialised() bool
+	shutdown() error
```

Comment:
`handlers_test.go` need to be modified as well the mock service to implement this method